cmd/dlv/dlv_test.go

```diff
@@ -849,11 +849,25 @@ func TestRemoteDAPClientAfterContinue(t *testing.T) {
 
 	go func() { // Capture logging
 		for scanErr.Scan() {
-			t.Log(scanErr.Text())
+			text := scanErr.Text()
+			if strings.Contains(text, "Internal Error") {
+				t.Error("ERROR", text)
```

Comment:
Is this just for visibility?

Reply:
Yes, otherwise it looked exactly like the rest of logging

Reply:
Ack, sgtm.